mlbstatsapi/mlbapi.py

```diff
@@ -132,7 +132,7 @@ def get_team_id(self, teamName) -> List[int]:
         
         if ('teams' in mlbdata.data and mlbdata.data['teams']):
             for team in mlbdata.data['teams']:
-                if team['teamName'] == teamName: # find matching Team Name
+                if team['teamname'] == teamName: # find matching Team Name
```

Comment:
Same thing here:
`if team['teamname'].lower() == teamName.lower(): # find matching Team Name`

Reply:
changed?

Wait I changed it it in the PR?